libs/blocks/tabs/tabs.js

```diff
@@ -57,6 +57,19 @@ export function getRedirectionUrl(linkedTabsList, targetId) {
   return currentUrl;
 }
 
+const loadActiveTabFromStorage = () => {
+  const activeTab = localStorage.getItem('active-tab');
```

Comment:
Is the requirement to make the tab selection persistent across visits? if it is only for the current navigation, we should be using sessionStorage.